tests/shell/BashateBearTest.py

```diff
@@ -0,0 +1,41 @@
+from bears.shell.BashateBear import BashateBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
```

Comment:
Ah and please add another blank line here too, the import section should be always separated with 2 blank lines

Reply:
(This was done)